quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -143,7 +146,10 @@
             DefaultJpaConnectionProviderFactory.class,
             DefaultLiquibaseConnectionProvider.class,
             FolderThemeProviderFactory.class,
-            LiquibaseJpaUpdaterProviderFactory.class);
+            LiquibaseJpaUpdaterProviderFactory.class,
+            DefaultHostnameProviderFactory.class,
+            FixedHostnameProviderFactory.class,
+            RequestHostnameProviderFactory.class);
```

Comment:
Should these be added here? Wasn't there supposed to be a single Quarkus specific provider?

Reply:
This is a list of providers being excluded. We only have the new one available at build/runtime.

Reply:
Ah, GitHub hid the IGNORED_PROVIDER_FACTORY part ;)

Reply:
Just btw: I like this a lot as a migration path whenever it comes to sth. we want to re-implement. :)